api/query/query_test.go

```diff
@@ -55,8 +56,14 @@ func TestLoadSummaries_success(t *testing.T) {
 		[]byte(`{"/x/y/z":[3,4]}`),
 	}
 	summaries := []summary{
-		map[string][]int{"/a/b/c": []int{1, 2}},
-		map[string][]int{"/x/y/z": []int{3, 4}},
+		{
+			Old: map[string][]int{"/a/b/c": {1, 2}},
```

Comment:
Rather than modifying the existing one, we should add new tests to test the `New` format

Reply:
Yes agreed, added a new test so that both the old and new formats are tested 👍